client/src/app/pages/checkout.js

```diff
@@ -55,6 +66,11 @@ class CheckoutPage extends Component {
         <section className={`checkout container ${onBlur}`}>
           <h1>Please enter your shipping information bellow:</h1>
           <CheckoutForm handleSubmit={this.handleSubmit} />
+          <h2>
+            Total Price:
+            {this.getTotalPrice(cart.productsInCart)}
```

Comment:
transform value to float (Number.toFixed(2)) or use:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/NumberFormat

Reply:
finished